### PR TITLE
vm: note on kvm dynamic scaling

### PR DESCRIPTION
**Pull request by shwstppr**
Related https://github.com/apache/cloudstack/issues/6626

### CODE REVIEW

**Comment by shwstppr** [comment]
@blueorangutan docbuild

File: source/adminguide/virtual_machines.rst [hunk 1]
```diff
@@ -493,6 +493,9 @@ Limitations
    issue with these operating systems, which will freeze if an attempt
    is made to dynamically scale from less than 3 GB to more than 3 GB.
 
+-  On KVM, not all versions of Qemu/KVM may support dynamic scaling and may result CPU or memory related failure during instance deployment.
```
Comment (DaanHoogland) [inline source/adminguide/virtual_machines.rst:496]:
```suggestion
-  On KVM, not all versions of Qemu/KVM may support dynamic scaling. Some combinations may result CPU or memory related failures during instance deployment.
```

**Comment by DaanHoogland** [comment]
@shwstppr this is in the chapter on vm instance creation. Is it required to put this in some prerequisites chapter? (This is a genuine question but it seems this remark is a bit after the fact on this location)

**Comment by DaanHoogland** [comment]
> > @shwstppr this is in the chapter on vm instance creation. Is it required to put this in some prerequisites chapter? (This is a genuine question but it seems this remark is a bit after the fact on this location)
> 
> @DaanHoogland can you please suggest the appropriate location where the note can be added?

I don´t know @shwstppr , It *is* a genuine question. I am thinking of the quick installation guide, the paragraph on KVM install (http://docs.cloudstack.apache.org/en/latest/quickinstallationguide/qig.html#kvm-setup-and-installation) or the Installation guide (http://docs.cloudstack.apache.org/en/latest/installguide/hypervisor/kvm.html#system-requirements-for-kvm-hypervisor-hosts)

It is certainly not wrong to mention it here, but also something I would like to know as an operator before choosing and installing hypervisors.

**Comment by shwstppr** [comment]
@DaanHoogland I've added a note in http://docs.cloudstack.apache.org/en/latest/installguide/hypervisor/kvm.html#system-requirements-for-kvm-hypervisor-hosts as well. I guess it is okay to have it at both places?

**Comment by DaanHoogland** [comment]
> @DaanHoogland I've added a note in http://docs.cloudstack.apache.org/en/latest/installguide/hypervisor/kvm.html#system-requirements-for-kvm-hypervisor-hosts as well. I guess it is okay to have it at both places?

yes, I agree

**Review by DaanHoogland — APPROVED** [review]

**Review by Pearl1594 — APPROVED** [review]
LGTM - may be worth adding error message that is seen on Qemu end that inidicates qemu's inability to support dynamic scaling

**Pull request merged**